src/main/java/org/geysermc/floodgate/mixin/ServerLoginPacketListenerMixin.java

```diff
@@ -0,0 +1,13 @@
+package org.geysermc.floodgate.mixin;
+
+import com.mojang.authlib.GameProfile;
+import net.minecraft.server.network.ServerLoginPacketListenerImpl;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(ServerLoginPacketListenerImpl.class)
+public interface ServerLoginPacketListenerMixin {
+
+    @Invoker("startClientVerification")
+    void startClientVerification(GameProfile profile);
```

Comment:
I can make this an access widener instead. I've initially left it as a mixin because it used to be one, and in case there are additional changes in the future that require mixins.

Reply:
I would say just do the access widener if that's the minimum needed to have it work.